pywren/serialize/cloudpickle/cloudpickle.py

```diff
@@ -158,17 +156,17 @@ def dump(self, obj):
 
     def save_memoryview(self, obj):
         """Fallback to save_string"""
-        Pickler.save_string(self, str(obj))
+        Pickler.save_str(self, str(obj)) # pylint: disable=no-member
```

Comment:
Good qn - so if you see the code paths below, we call `save_memoryview` only for Python 3. And if you look at the `_Pickler` implementation in cpython [1] then you can see that the method is called `save_str`. The same file in the 2.7 branch [2] uses `save_string`

[1] https://github.com/python/cpython/blob/3.5/Lib/pickle.py#L698
[2] https://github.com/python/cpython/blob/2.7/Lib/pickle.py#L486